gui/control.js

```diff
@@ -71,6 +95,16 @@ function render() {
             showToast("Error!", errorToMessage[error])
         });
     }
+
+    const userPictures = document.querySelectorAll(".user-picture");
+    userPictures.forEach(function(picture) {
+        const username = picture.dataset.username;
+        picture.textContent = getInitials(username);
+    });
+
+    function getInitials(name) {
+        return name.split(" ").map(word => word[0]).join("").toUpperCase();
+    }
```

Comment:
maybe move out of the render function, or in some util file as the function is generic